scripts/add_existing_baseyear.py

```diff
@@ -254,7 +258,11 @@ def add_power_capacities_installed_before_baseyear(n, grouping_years, costs, bas
         name_suffix = f" {generator}{suffix}-{grouping_year}"
         name_suffix_by = f" {generator}{suffix}-{baseyear}"
         asset_i = capacity.index + name_suffix
-        if generator in ["solar", "onwind", "offwind"]:
+        if generator in ["solar", "onwind", "offwind-ac"]:
+            if generator == "offwind-ac":
+                cost_key = "offwind"
+            else:
+                cost_key = generator
```

Comment:
```suggestion
        cost_key = generators.split("-")[0]
```
This should also work and is simpler.

Reply:
I need the line `if generator in ["solar", "onwind", "offwind-ac"]` to iterate over renewables only.